### ISSUE TITLE
Website UX issue nightly builds and other content hard to find

### ISSUE THREAD

**Issue by tim-seoss** [memtest86plus/memtest86plus#455]
The README says:

"Binary releases (both stable and nightly dev builds) are available on [memtest.org](https://memtest.org/)."

... but I can't see any sign of them on the download section of the website. I've built from source without problems, but the website or README need updating I think?

**Comment by martinwhitaker** [comment]
I've just looked and they are still there. Stable builds are on the main page (https://memtest.org). Scroll down past the FAQ. Nightly builds are on the dev page (https://memtest.org/dev) - accessed via the rather obscure `</>` icon on the top bar

**Comment by tim-seoss** [comment]
Thanks.  Finding the Stable builds were no-problem, it was the nightlies that I failed to find... I've changed the bug title accordingly. Assuming the site layout is to be retained then I think that icon probably needs changing?  Perhaps just the text "Source and Nightlies"?  Alternatively it could be "Dev".

Might also be worth linking to the /dev page from the stable download tile (especially if `</>` is changed to something like which doesn't explicitly mention the nightlies?
